server/app/routes/index.js

```diff
@@ -11,6 +11,10 @@ router.use('/v1/songs', require('./songs'));
 
 router.use('/v1/cart', require('./cart'));
 
+router.use('/v1/guest', require('./guest'));
+
+router.use('/v1/order', require('./order'));
```

Comment:
line 16 of server/app/routes/index.js - checkout orders needs to be synced with #109